code-samples/ponypath-unix-mac.sh

```diff
@@ -0,0 +1 @@
+export PONYPATH=$PONYPATH:$HOME/pony
```

Comment:
Is there benefit in having these non-Pony-source-code samples externalized?

Maybe I'm missing something but I wouldn't expect us to get much benefit from externalizing them if we're not going to compile/test them.

Reply:
Yeah, I don't see any, either. I just went for consistency, but if you want me to revert these, I can do so, accordingly.

Reply:
Discussed on the sync call today. We decided we'd prefer not to externalize these.

Can you please move the non-pony source files back into the main markdown?

Reply:
Will do 👍🏻